src/screens/Leaderboard/Leaderboard.tsx

```diff
@@ -0,0 +1,64 @@
+import { FC } from 'react';
+import { ArrowDownIcon, ArrowUpIcon, MinusIcon } from '@heroicons/react/24/outline';
+
+import Card from '../../components/card/card.tsx';
+
+type Leader = {
+  name: string;
+  delta: number;
+};
+
+interface LeaderboardProps {
+  leaders: Leader[];
+}
+
+export const Leaderboard: FC<LeaderboardProps> = ({ leaders }) => {
+  const getDeltaInfo = (delta: number) => {
```

Comment:
It seems like we re-initialising this function on every re-render. How about wrapping it in useCallback 🤔